### ISSUE TITLE
Feature: Add z-index values to report

### ISSUE THREAD

**Issue by davetgreen** [katiefenn/parker#43]
I think this would be useful to see if you've ended up with ridiculous z-indexes! :+1:

**Comment by vincentorback** [comment]
Not sure how to list these tho. Show highest? Show total?
I have [a little script here](https://gist.github.com/vincentorback/1dbf671de067589bc48869969e929e15) that lists all z-index properties, which can be a good thing to once in a while. But maybe it’s a but much for this plugin?

**Comment by bartveneman** [comment]
Or show all unique values, like here? (personal very early side project): https://projectwallace.herokuapp.com/users/1/projects/8/metrics/unique_zindexes

**Comment by davetgreen** [comment]
@vincentorback @bartveneman Personally I'd like to see the highest value and the total number of unique values. I don't think anything else is really all that useful. If you see that you've got high figure of 9999 or you have 150 (for example) different values then you know that's silly and you need to intervene.

**Comment by bartveneman** [comment]
That sounds reasonable, although I've seen some projects using groupings like 100-103 for site headers, 200-204 for some component and 900-910 for modals. As always, I guess: it depends

**Comment by vincentorback** [comment]
Yeah, it’s a good thing to see all of your z-index's to see that they stack with a reasonable amount. But I’m not sure it’s **parkers** job to show this. So maybe showing highest and total is good enough!

**Comment by bartveneman** [comment]
> Personally I'd like to see the highest value and the total number of unique values.

> But I’m not sure it’s parkers job to show this. So maybe showing highest and total is good enough!

I think you're both right. Parker usually shows averages/means, so deviating from that would be indeed a little strange.
